content/cpp/concepts/smart-pointers/smart-pointers.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: 'Smart Pointers'
+Description: 'A smart pointer is a class that holds and manages a pointer through RAII.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Smart Pointers'
+  - 'Pointers'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+A smart pointer is a class that holds and manages a pointer through RAII. It allows developers to not worry about freeing a pointer, and it also allows pointers to be exception-safe.
```

Comment:
For readers who aren't too familiar with what "RAII" is, can we expand on the first mention and use "RAII" for the rest of the entry?

For instance, 

```markdown
A smart pointer is a class that holds and manages a pointer through a popular C++ technique called "Resource acquisition is initialization" (**RAII**).
```